src/Gitonomy/Git/Parser/DiffParser.php

```diff
@@ -25,7 +25,7 @@ protected function doParse()
 
         while (!$this->isFinished()) {
             // 1. title
-            $vars = $this->consumeRegexp('/diff --git (a\/.*) (b\/.*)\n/');
+            $vars = $this->consumeRegexp('/diff --git "?(a\/.*)"? "?(b\/.*)"?\n/');
```

Comment:
Can you add a test for this?

Reply:
While I completly agree with you that this case needs to be tested - I am not sure how to test it since the tests seem to depend on the foobar-repository. 
Should I clone the repository and add a file which I rename?

Reply:
Yes 👍🏼

Reply:
I made a [pull request ](https://github.com/gitonomy/foobar/pull/2) and ready to write tests :)

Reply:
I merged your PR :) thanks